aries-site/src/tests/examples/screen-container.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-undef */
+import { Selector } from 'testcafe';
+import { waitForReact } from 'testcafe-react-selectors';
+import { baseUrl, takeResponsiveSnapshots } from '../utils';
+
+fixture('Screen Container')
+  .page(`${baseUrl}/templates/wizard`)
+  .beforeEach(async () => {
+    await waitForReact();
+  });
+
+test('should wrap content, content should fill width on laptop', async t => {
+  await takeResponsiveSnapshots(t, 'Screen Container on Laptop');
+});
+
+test(`should wrap content, content should fill partial width on 
+desktop`, async t => {
+  const desktopButton = Selector('button').withText('Desktop');
+  await t.click(desktopButton);
+  await takeResponsiveSnapshots(t, 'Screen Container on Desktop');
+});
+
+test('should not appear on mobile', async t => {
```

Comment:
The name of this test confuses me. What should not appear on mobile? The mock browser wrapper? How about the content?

Reply:
Good question, since the fixture is called `Screen Container`, then it will print out in the test runner as:
```
Screen Container
   should not appear on mobile
   name of other test
   name of another thest
```
but I can see how just reading line 23 is ambiguous. How about we change it to "should not display mock browser for mobile display"?